application/models/item.go

```diff
@@ -928,14 +930,21 @@ func (i *Item) CalculateBillingPremium(tx *pop.Connection) api.Currency {
 // CalculateAnnualPremium returns the rounded product of the item's CoverageAmount and the category's
 // PremiumFactor, with the category's minimum premium applied
 func (i *Item) CalculateAnnualPremium(tx *pop.Connection) api.Currency {
+	i.LoadCategory(tx, false)
+	factor := i.GetPremiumFactor(tx)
+	premium := int(math.Round(float64(i.CoverageAmount) * factor))
+
+	return api.Currency(domain.Max(premium, i.Category.MinimumPremium))
```

Comment:
Let's ditch the domain.Max use as we are on go 1.22 now, so we can use the builtin `max` function instead.

```suggestion
	return api.Currency(max(premium, i.Category.MinimumPremium))
```